packages/entity-list/src/components/Table/Table.js

```diff
@@ -80,7 +80,7 @@ const Table = props => {
       singleSelectHandler(entity.__key, true, true)
     } else if (e.metaKey || e.ctrlKey) {
       singleSelectHandler(entity.__key)
-    } else {
+    } else if (props.clickable !== false) {
```

Comment:
you have to rebase. i added a check for "onRowClick defined" recently

Reply:
this PR is for 2.26 unfortunately. in the master branch, this file looks quite different. i will resolve the conflicts and reapply this clickable check when merging the 2.26 branch into the master branch.